hack/test

```diff
@@ -60,6 +60,8 @@ function unittest_pass {
 			rm profile.out
 		fi
 	done
+	# Send reports to codecov, CODECOV_TOKEN env must be present in Jenkins
+	curl -s https://codecov.io/bash | bash
```

Comment:
The codecov bash uploader will output an HTTP 400 error message but the unit test output script is not affected and continues as normal to either print `test success ===` or `test fail ===` based on the unit tests.

Or I can explicitly check for the presence of $CODECOV_TOKEN and not call the codecov script if that seems better.